image_recognition_openpose/src/image_recognition_openpose/openpose_wrapper.py

```diff
@@ -45,7 +45,10 @@ def __init__(self, model_folder, pose_model, net_input_size, net_output_size, nu
 
         logging.info("Loading openpose with parameters: %s", parameters)
 
-        self._openpose = openpose.OpenPose(parameters)
+        self._op = op
```

Comment:
Should not be a member variable

Reply:
Openpose is imported in the constructor, so it will go out of scope. @ar13pit, this might be a sotution: https://stackoverflow.com/questions/11990556/how-to-make-global-imports-from-a-function

Reply:
I don't like globals. Instead of using global, I would rather move the import out of the constructor and let the user make sure pyopenpose is installed.

Reply:
@MatthijsBurgh the solution suggested in stackoverflow is a bad one, as per the docs of `global` in python:
https://docs.python.org/3/reference/simple_stmts.html#the-global-statement

Reply:
And the second solution: https://stackoverflow.com/a/11991605/12135039

Reply:
That's a better one and a legal operation in python.